packages/neuron-wallet/electron-builder.yml

```
@@ -1,6 +1,7 @@
 appId: com.nervos.neuron
 copyright: Copyright (C) 2019 Nervos Foundation. All rights reserved
 productName: Neuron
+artifactName: "${productName}-${version}.${ext}"
```

I suggest we also include platform in the name so users know which file to download for their OS.

```suggestion
artifactName: "${productName}-${version}-${os}.${ext}"
```

---

BTW for Linux and Windows, it might also be a good idea to append `${arch}` as the last part (after `${os}`).